impl/client/src/main/javascript/web/vizapi/ccc/ccc_wrapper.js

```diff
@@ -3707,131 +3707,74 @@ define([
         },
 
         _processSelection: function(selectedDatums){
-            if(selectedDatums.length){
-                var data = this._chart.data;
-
-                var atomsByDim = {};
+            var whereSpec;
```

Comment:
As far as I was able to detect, "diffing" by eye, the new code of this method is exactly the same as the code of the same method, `_processSelection`, of the root class, `pentaho.ccc.Chart`. So why copy & paste and not deleting the override? Or maybe I missed a difference.